### PR TITLE
SAK-42904 Missing documentation for property for disabling duplicate email addresses for accounts

### PR DESCRIPTION
**Pull request by mpellicer**
https://jira.sakaiproject.org/browse/SAK-42904

### CODE REVIEW

File: config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties [hunk 1]
```diff
@@ -1112,7 +1112,9 @@
 # Default: You have entered an email address ending with "{1}", which is restricted. Please try again with a different email address.
 # user.email.invalid.domain.message=You have entered a {0} email address ending with "{1}". Please log in with your existing {0} account.
 
-
+# Allow users with duplicated email
```
Comment (fostersdesign) [inline config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties]: Let's change this descriptive line to:
"Allow users to create new accounts with email addresses already being used (a duplicate email address)"

**Comment by fostersdesign** [comment]
@ern, how was this merged without approval? I had requested changes.

**Comment by ern** [comment]
I implemented them and merged!

**Comment by fostersdesign** [comment]
Oh good! Thanks! I didn't see the change listed on the Files tab, so I got worried.

**Pull request merged**